src/dtemplate.d

```diff
@@ -1,10 +1,14 @@
-// Compiler implementation of the D programming language
-// Copyright (c) 1999-2016 by Digital Mars
-// All Rights Reserved
-// written by Walter Bright
-// http://www.digitalmars.com
-// Distributed under the Boost Software License, Version 1.0.
-// http://www.boost.org/LICENSE_1_0.txt
+/**
+ * Compiler implementation of the
+ * $(LINK2 http://www.dlang.org, D programming language).
+ *
+ * Template implementation.
+ *
+ * Copyright:   Copyright (c) 1999-2016 by Digital Mars, All Rights Reserved
```

Comment:
If you already do this cleanup, may I ask a dumb question as I don't know much about the history: shouldn't we start to license as much as possible to the D Foundation?